chapter03/qli/src/test/java/spring/training/service/SpringBankServiceTest.java

```diff
@@ -20,10 +22,14 @@
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = { "classpath:test.xml" })
+@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
 public class SpringBankServiceTest {
```

Comment:
No new tests to check for auditing?
I am curious how people are going to test that.. I just added a counter ;)